test/asyncapi_test.go

```diff
@@ -40,6 +40,29 @@ func (s *CLITestSuite) TestAsyncApiExport() {
 			s.Error(nil, "spec generated does not match the template output file")
 		}
 	}
+	resetConfiguration(s.T(), false)
+}
 
+func (s *CLITestSuite) TestAsyncApiImport() {
+	tests := []CLITest{
+		//Input file not specified
+		{args: "asyncapi import", exitCode: 1},
+		//No Kafka selected
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml", exitCode: 1},
+		//No SR Key setup
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml", exitCode: 1, useKafka: "lkc-asyncapi", authKafka: "true"},
+		{args: "environment use " + testserver.SRApiEnvId, workflow: true},
+		// Overwrite=false
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml --schema-registry-api-key ASYNCAPIKEY --schema-registry-api-secret ASYNCAPISECRET", useKafka: "lkc-asyncapi", authKafka: "true", workflow: true, fixture: "asyncapi/2.golden"},
+		//Overwrite=true
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-spec.yaml --schema-registry-api-key ASYNCAPIKEY --schema-registry-api-secret ASYNCAPISECRET --overwrite=true -vvv", useKafka: "lkc-asyncapi", authKafka: "true", workflow: true},
+		//input file with 0 channels
+		{args: "asyncapi import ./test/fixtures/input/asyncapi/asyncapi-with-context.yaml --schema-registry-api-key ASYNCAPIKEY --schema-registry-api-secret ASYNCAPISECRET --overwrite=true -vvv", useKafka: "lkc-asyncapi", authKafka: "true", workflow: true},
```

Comment:
The commands with `workflow: true` should be in a separate function as the commands that are not part of the workflow.